packages/wallet-sdk/src/sign/util.ts

```diff
@@ -17,19 +22,26 @@ export async function storeSignerType(signerType: SignerType): Promise<void> {
   return storage.setItem(SIGNER_TYPE_KEY, signerType);
 }
 
+export type ScwOnboardMode = 'default' | 'create';
```

Comment:
I think we can remove this type from the SDK

Reply:
cc @nateReiners